lib/rouge/lexers/liquid.rb

```diff
@@ -0,0 +1,275 @@
+# -*- coding: utf-8 -*- #
+
+module Rouge
+  module Lexers
+    class Liquid < RegexLexer
+      title "Liquid"
+      desc 'Liquid is a templating engine for Ruby (liquidmarkup.org)'
+      tag 'liquid'
+      filenames '*.liquid'
+
+      def test(val)
+        mixin :generic
+      end
+
+      state :root do
+        rule /[^\{]+/, Text
+        rule /(\{%)(\s*)/ do
+          groups Punctuation, Text::Whitespace
+          push :tagBlock
+        end
+        rule /(\{\{)(\s*)/ do
+          groups Punctuation, Text::Whitespace
+          push :output
+        end
+        rule /\{/, Text
+      end
+
+      state :tagBlock do
```

Comment:
The convention here is to use `snake_case` for symbols, if you could convert these that'd be awesome.